blueman/main/Tray.py

```diff
@@ -0,0 +1,44 @@
+from importlib import import_module
```

Comment:
If we do then we should add a follow up commit to change configure.ac. We can then also drop the major/minor python version bits.